epam-promo/components/types.ts

```diff
@@ -50,10 +50,11 @@ export interface RowSizeMod {
 
 export interface IHasEditMode {
     /** Defines input mode: 'form' is default and usual look, 'cell' is for using components as table cells  */
-    mode?: 'form' | 'cell';
+    mode?: 'form' | 'cell' | 'inline';
```

Comment:
don't add 'inline' mode in promo, becuse it will work only for component from @epam/uui, leave it only in @epam/uui, after we move all components to the @epam/uui, we will remove this types from skins